File: java/MaratonaJava/javinha/src/main/java/com/maratonajava/estrutura_sequencial/exerc05.java
package com.maratonajava.estrutura_sequencial;

import java.util.Scanner;

public class exerc05 {
    //Faça um Programa que converta metros para centímetros.
    public static void main(String[] args){
        try(Scanner convert = new Scanner(System.in)){
            float metro, convertCent;

            System.out.println("Programa Converter Metros para Centímetros.");
            System.out.println("================================================");

            System.out.println("Informe o valor em metros.");
            metro = convert.nextFloat();

            convertCent = metro*100;

            System.out.println("O Tamanho em centímetros "+convertCent+" cm.");

        }
    }
}

File: C/Exercicios/Estrutura_Sequencial/exerc05.c
#include <stdio.h>
#define descricao_programa "Faça um Programa que converta metros para centímetros."

int main(){
    printf("%s\n",descricao_programa);

    float  metros, conversao;

    printf("Informe um valor em Metros.\n");
    scanf("%f",&metros);

    conversao = metros*100;

    printf("O Valor em Centimétro é:%f.\n", conversao);

}